packages/site/src/utils/snap.ts

```diff
@@ -67,4 +68,53 @@ export const sendHello = async () => {
   });
 };
 
+/**
+ * Toggle paymaster usage.
+ */
+
+export const togglePaymasterUsage = async () => {
+  await window.ethereum.request({
+    method: 'wallet_invokeSnap',
+    params: {
+      snapId: defaultSnapOrigin,
+      request: { method: 'snap.internal.togglePaymasterUsage' },
+    },
+  });
+};
+
+export const isUsingPaymaster = async (): Promise<boolean> => {
+  return (await window.ethereum.request({
+    method: 'wallet_invokeSnap',
+    params: {
+      snapId: defaultSnapOrigin,
+      request: { method: 'snap.internal.isUsingPaymster' },
+    },
+  })) as boolean;
+};
+
+export const isCurrentChainConfigured = async (): Promise<boolean> => {
+  const currentChainId = (await window.ethereum.request({
+    method: 'eth_chainId',
+  })) as string;
+  const configs = (await window.ethereum.request({
+    method: 'wallet_invokeSnap',
+    params: {
+      snapId: defaultSnapOrigin,
+      request: { method: 'snap.getConfigs' },
```

Comment:
I believe it should be `'snap.internal.getConfigs'` see [here](https://github.com/MetaMask/snap-account-abstraction-keyring/blob/fix/chain-config-setup/packages/snap/src/permissions.ts#L7)